playwright/package.json

```diff
@@ -8,12 +8,12 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "@playwright/test": "^1.35.0",
-    "@types/node": "^20.6.0",
-    "@types/uuid": "^9.0.3",
-    "typescript": "^5.2.2"
+    "@playwright/test": "^1.43.0",
+    "@types/node": "^20.12.6",
```

Comment:
Ideally this should match the version of node. We're using 16.x in the pipelines.